awx/ui/src/components/Schedule/shared/ScheduleForm.js

```diff
@@ -100,6 +100,24 @@ function ScheduleFormFields({ hasDaysToKeepField, zoneOptions }) {
   });
   const [{ name: dateFieldName }] = useField('startDate');
   const [{ name: timeFieldName }] = useField('startTime');
+  const [timezoneMessage, setTimezoneMessage] = useState('');
+  const warnLinkedTZ = (event, selectedValue) => {
+    if (zoneLinks[selectedValue]) {
+      setTimezoneMessage(
+        `Warning: ${selectedValue} is a link to ${zoneLinks[selectedValue]} and will be saved as that.`
```

Comment:
To mark this string for translation you'll need to import the `t` macro from `lingui` library and wrap it, e.g. 

```
t`my string for translation`
```